### ISSUE TITLE
Issues that may Occur.

### ISSUE THREAD

**Issue by DenkSnek** [Topaz-Reality/D-Trainer#5]
*I ran this section with a save file that already had a decent amount of mods activated, so this part is just my mistake. It wasn't wise to run it like that and I am currently going through it without mods activated.*

1.) I was curious and, for some reason, decided to run both of them at the same time. Things got crazy real quick. 
https://gyazo.com/0866464781876fa0090f88b0c4fbb923
https://gyazo.com/23e72ee9b09cca03e5b950e120359a3b
Ultima Weapon was equipped and Donald was no where to be seen lol. I started moving around and the screen just got messed up, as in the screenshots. It was quite the experience.

*After playing through without mods, it seems that it doesn't roll the same way as I just mentioned. Rather, it just crashes immediately when transitioning between rooms.*

2.) For Stormfall, it seems that I crash after entering a room. It shows the transition that opens up the room, but it crashes early into it. I did it within 2 seconds and I didn't do it as i entered the room, so I don't know. This was the result of the scenario without extra mods. With the extra mods, I was able to get into the next room without it crashing, but Aqua's model would appear for a split second and Ultima Weapon would show along with Stormfall as if he was holding two in one hand.
https://gyazo.com/059e6b06defcdded5acc510680db85b0

3.) As for Roxas, some weird shit happens. The screen goes black upon transition and stays black throughout the room. I can still play it, as I hear sounds and I can open the menu, Although, if I open the menu and change keyblades, it crashes. Also, if I go into the menu and unpause, the screen does this. 
https://gyazo.com/e3a15f8b61bf69777f375192538a40da
I genuinely have no idea what I did, but I know for sure that I messed something up, because it just crashes no matter what. I know that because, at first, it worked beautifully. However, now it just crashes. I don't know what I did. The only other issue that I encountered with Roxas, is that drive forms cause the game to crash, which I believe you mentioned in another post.

I don't know, I'm just trying to point out some things that went wrong when I tried it out. It could be on my end that these issues occur, or it could be the Trainer. I don't know, I'm inept in this field and I apologize if I wasted your time. 

Also, this was just based on a quick time period. Later, I'll do a natural playthrough without mods and I'll try to mention other issues that occur. This is a really cool trainer and I can't wait to see what else comes.

**Comment by Topaz-Reality** [comment]
Now, Let's clear things up.

Stormfall and Roxas: Cannot use them both at the same time. Because they both rely on Winnie the Pooh which is replaced by Donald.

Stormfall: Within 2 second doesn't mean AFTER 2 seconds. It means immediately or 1 second after the activation. Because if Aqua doesn't replace Donald, how is it supposed to adjust it?

Roxas: Umm... What the living hell?

All mods: All of the mods crash if a Drive Form is used, because of the intense texture appliance.  Will try to fix. For Aqua's Keyblade, Just close the trainer and transform, then activate it again (Will crash on de-transform)

Don't use this with HDification. It will cause memory runouts.

**Comment by DenkSnek** [comment]
Oh okay, that makes a LOT more sense. Thank you very much. As I said before, I'll mess around with it later and report on more issues.

Also, I think I figured out what I was doing wrong for Stormfall. I was at Port Royal and I was transitioning rooms to the ship deck, where the party members aren't party members. I just tried it out in Hollow Bastion and it didn't crash. It still looks like he's dual wielding, though.
